
---------------------------------- log of pk 18 : 2020-03-21 12:08:05.141681 ----------------------------------
출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

에러 메시지:
-----------------------------------------------------------------------------------------------------------

---------------------------------- log of pk 19 : 2020-03-21 13:18:07.601558 ----------------------------------
출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

에러 메시지:
-----------------------------------------------------------------------------------------------------------

    ---------------------------------- log of pk 20 : 2020-03-21 14:53:39.194545 ----------------------------------
    출력 메시지:

    에러 메시지:
  File "./work/20.py", line 73
    prec = "용산구 갑
                 ^
SyntaxError: EOL while scanning string literal
    -----------------------------------------------------------------------------------------------------------
    
    ---------------------------------- log of pk 21 : 2020-03-21 14:58:17.984887 ----------------------------------
    출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

    에러 메시지:
    -----------------------------------------------------------------------------------------------------------
    
    ---------------------------------- log of pk 21 : 2020-03-21 15:00:39.643887 ----------------------------------
    출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

    에러 메시지:
    -----------------------------------------------------------------------------------------------------------
    
    ---------------------------------- log of pk 21 : 2020-03-21 15:01:08.016509 ----------------------------------
    출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

    에러 메시지:
    -----------------------------------------------------------------------------------------------------------
    
    ---------------------------------- log of pk 21 : 2020-03-21 15:14:36.542664 ----------------------------------
    출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

    에러 메시지:
    -----------------------------------------------------------------------------------------------------------
    
    ---------------------------------- log of pk 21 : 2020-03-21 15:19:42.144303 ----------------------------------
    출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

    에러 메시지:
    -----------------------------------------------------------------------------------------------------------
    
---------------------------------- log of pk 22 : 2020-03-21 17:57:20.467509 ----------------------------------
출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

에러 메시지:
-----------------------------------------------------------------------------------------------------------

---------------------------------- log of pk 24 : 2020-03-21 18:05:22.189400 ----------------------------------
출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

에러 메시지:
-----------------------------------------------------------------------------------------------------------

---------------------------------- log of pk 25 : 2020-03-21 18:12:26.482864 ----------------------------------
출력 메시지:

에러 메시지:
  File "./work/25.py", line 73
    prec = "금천구" 갑"
                 ^
SyntaxError: invalid syntax
-----------------------------------------------------------------------------------------------------------

---------------------------------- log of pk 26 : 2020-03-21 23:08:38.752728 ----------------------------------
출력 메시지:
Requirement already up-to-date: pip in ./dataVenv/lib/python3.8/site-packages (20.0.2)

에러 메시지:
-----------------------------------------------------------------------------------------------------------
